src/controller/storagemgr/storagedriver/storagehandler_test.go

```diff
@@ -1,19 +1,19 @@
-/*******************************************************************************
- * Copyright 2020 Samsung Electronics All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- *******************************************************************************/
+// Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved.
```

Comment:
Please swap the first and second lines.
Don't put a blank line between copyright.
```suggestion
//
// Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved.
```

Reply:
Thanks for the Review, Just to be clear : 

//
// Samsung (c)
// Intel (c)
// Same as before 

Is this correct ?